components/script/dom/window.rs

```diff
@@ -2018,9 +2018,10 @@ impl Window {
         if self.prepare_for_screenshot && for_display {
             // Checks if the html element has reftest-wait attribute present.
             // See http://testthewebforward.org/docs/reftests.html
+            // and https://web-platform-tests.org/writing-tests/crashtest.html
             let html_element = document.GetDocumentElement();
             let reftest_wait = html_element.map_or(false, |elem| {
-                elem.has_class(&atom!("reftest-wait"), CaseSensitivity::CaseSensitive)
+                elem.has_class(&atom!("reftest-wait"), CaseSensitivity::CaseSensitive) || elem.has_class(&atom!("test-wait"), CaseSensitivity::CaseSensitive)
```

Comment:
I could also dublicate this:
```
            let test_wait = html_element.map_or(false, |elem| {
                elem.has_class(&atom!("test-wait"), CaseSensitivity::CaseSensitive)
            });
            ```

Reply:
Now that we have a prefs file that is only applied during WPT tests, could we gate the class name checks on a new preference and set that preference in the file?

Reply:
Actually we can leave that for a followup issue. This seems fine to merge if we use Atom::from instead of the macro.

Reply:
> Now that we have a prefs file that is only applied during WPT tests, could we gate the class name checks on a new preference and set that preference in the file?

Could you elaborate on this? I didn't quite understand, and I would like to follow up.